smetrics/src/main/scala/com/evolutiongaming/smetrics/MeasureDuration.scala

```diff
@@ -59,4 +59,33 @@ object MeasureDuration {
       val start = f(self.start.map(f.apply))
     }
   }
+
+  object implicits {
```

Comment:
Mb pull this out to the top level, so it would be `import com.evolutiongaming.smetrics.implicits._`

Reply:
will do imports similar to Cats